test/setup/main.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 3.0"
+
+  name              = "ci-log-export"
+  random_project_id = "true"
+  org_id            = var.org_id
+  folder_id         = var.folder_id
+  billing_account   = var.billing_account
+
+  activate_apis = [
+    "cloudresourcemanager.googleapis.com",
+    "oslogin.googleapis.com",
+    "serviceusage.googleapis.com",
+    "compute.googleapis.com",
+    "bigquery-json.googleapis.com",
+    "pubsub.googleapis.com",
+    "storage-component.googleapis.com",
+    "storage-api.googleapis.com",
+    "logging.googleapis.com",
+    "iam.googleapis.com",
+    "cloudbilling.googleapis.com"
+  ]
+}
+
+resource "null_resource" "wait_apis" {
```

Comment:
I'm not convinced this is the same provider issue, but fine with keeping this for now. Opened #34 to track.

Reply:
Well, I got intermittent build failures because tests were failing on storage API not enabled, while the API was just enabled during the setup phase. I'm assuming this should be related. But I'm not sure if we should report the wait for API issue to the provider upstream as different to wait for permissions which is already logged.